src/index.html

```diff
@@ -1,16 +1,45 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <form
+      class="search-form"
+      data-qa="big"
+      >
+        <label class="search-label">
+          <input
+          class="search-input search-big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;" data-qa="keypress"
+          >
+        </label>
+      </form>
+      <div class="space"></div>
+      <form
+      class="search-form"
+      data-qa="small"
+      >
```

Comment:
```suggestion
      <form
        class="search-form"
        data-qa="small"
      >
```

![image](https://user-images.githubusercontent.com/80700247/224737782-4ba20542-a771-4800-813c-cf9aae9d4dfc.png)

Fix all such cases